### PR TITLE
User-profile frontend

### PR DESCRIPTION
**Pull request by ayeshaqaisar**
User-profile Frontend
Screenshot (1099)
Screenshot (1100)
Screenshot (1101)

### CODE REVIEW

**Review by Devanshi-123 — APPROVED** [review]

**Review by Devanshi-123 — CHANGES_REQUESTED** [review]
1. Make the image section round instead of the square as shown in the design
2. Export this file  user.js and import it in app.js and use that component in app.js

**Comment by raksh543** [comment]
@ayeshaqaisar Also please do proper formatting of your files (indentation and remove extra lines)

File: client/public/index.html [hunk 1]
```diff
@@ -24,6 +24,14 @@
...
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./style.css">
+    <link
+    rel="stylesheet"
+    href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300"/>
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```
Comment (divyaprakash-Rx) [inline client/public/index.html:34]: @ayeshaqaisar Please remove these and use npm modules instead.

File: client/public/index.html [hunk 2]
```diff
@@ -39,5 +47,12 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+
+
+    <!-- FOR BOOTSTRAP -->
+  <script src="https://code.jquery.com/jquery-3.5.1.min</script>.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
```
Comment (divyaprakash-Rx) [inline client/public/index.html:55]: @ayeshaqaisar Please remove these and use npm modules instead.

**Review by divyaprakash-Rx — CHANGES_REQUESTED** [review]
@ayeshaqaisar As your page contains many small components repeated multiple times, please make their single components and import after that.
e.g. you can create profile-card.js and use it again again passing names and img url as props.

**Comment by ayeshaqaisar** [comment]
> 1. Make the image section round instead of the square as shown in the design
> 2. Export this file  user.js and import it in app.js and use that component in app.js

Screenshot (1112)
Changed the image's shape
and imported user.js in app.js

**Review by divyaprakash-Rx — CHANGES_REQUESTED** [review]
@ayeshaqaisar  Great Job 🎉!
You have made many of your parts responsive and code structure is also clean now.
But these are some places that requires your attention:-
- [x] The Profile picture keeps on floating and '+' button is not fixed with it. 
One of the fixes for it is wrap both of them inside a div and then use it as a single component(e.g. Profilepic); and then wrap the complete part(Profilepic and name&organization div(class=User-profile-top) together. 

Also, it is of a fixed size due to which it goes out of view box in mobile views.
- [ ] The div inside which there are inputs for name and organization, is of completely fixed size due to which it goes out of viewbox for screen sizes smaller than 800px. 
One of the fixes for this is to make that div of 50vw for screen size greater than 768px and for screen sizes below 768px, make it to 100vw.
- [ ] Please make input box of activity of flexible size. It also goes out of viewbox.

I have added some of screenshots for your reference.

FireShot Capture 062 - React App - deploy-preview-9--muckin netlify app
Screen Shot 2020-10-24 at 15 13 01
Screen Shot 2020-10-24 at 15 13 08
Screen Shot 2020-10-24 at 15 15 01

**Review by divyaprakash-Rx — APPROVED** [review]
Perfect!✨🎉

**Review by Devanshi-123 — APPROVED** [review]

**Review by raksh543 — CHANGES_REQUESTED** [review]
@ayeshaqaisar Please solve these alignment issues.
image

- Make the edit and bin buttons clickable (cursor: pointer).
- On clicking on +, user should be able to browse image from the device.

**Comment by ayeshaqaisar** [comment]
> @ayeshaqaisar Please solve these alignment issues.
> image
> 
> * Make the edit and bin buttons clickable (cursor: pointer).
> * On clicking on +, user should be able to browse image from the device.

Done. Please check

**Review by raksh543 — APPROVED** [review]

**Pull request merged**